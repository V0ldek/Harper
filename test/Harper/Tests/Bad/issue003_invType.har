main :: Integer;
main = g 42;

g :: Integer -> Integer;
g n = f n "Str";

f :: Integer -> Integer -> Integer;
f x y = x * y;
