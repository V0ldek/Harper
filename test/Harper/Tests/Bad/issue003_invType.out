Error: expression `"Str"` has invalid type 'String'; expected 'Integer'.
During evaluation of: 
 f n "Str"
Located at line 5 column 7

Execution terminated with an error: type error.